Connectors/Revit/Speckle.Connectors.RevitShared/Plugin/IRevitIdleManager.cs

```diff
@@ -1,14 +0,0 @@
-namespace Speckle.Connectors.Revit.Plugin;
-
-// POC: needs interface
-// is probably misnamed, perhaps OnIdleCallbackManager
-internal interface IRevitIdleManager
-{
-  /// <summary>
-  /// Subscribe deferred action to Idling event to run it whenever Revit becomes idle.
-  /// </summary>
-  /// <param name="action"> Action to call whenever Revit becomes Idle.</param>
-  /// some events in host app are trigerred many times, we might get 10x per object
-  /// Making this more like a deferred action, so we don't update the UI many times
```

Comment:
double check if this documentation needs copying over

Reply:
this is moved